<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simple ARX</string>
  </property>
  <widget class="QPushButton" name="btnRun">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>530</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Run Arx Demo</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="txtInput">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>291</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="txtOutput">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>311</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpen">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>460</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblInput">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>271</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>input file ...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>450</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(138, 226, 52)
</string>
   </property>
   <property name="text">
    <string>Save Result</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
